NuRadioReco/framework/station.py

```diff
@@ -261,33 +261,37 @@ def get_magnetic_field_vector(self, time=None):
             logger.error(f"Reference reconstruction not set / unknown: {self.__reference_reconstruction}")
             raise ValueError(f"Reference reconstruction not set / unknown: {self.__reference_reconstruction}")
 
-    def avg_SNR(self, channels_to_include = [0, 1, 2, 3, 5, 6, 7, 22, 23]):
+    def get_average_signal_to_noise_ratio(self, channels_to_include = [0, 1, 2, 3, 5, 6, 7, 22, 23]):
```

Comment:
Can you set the default value for `channels_to_include` to `None`? This class should be a rather generic class and the channels id only make sense for RNO-G.